packages/terra-date-picker/src/DatePicker.module.scss

```diff
@@ -1,82 +1,240 @@
-/* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+// stylelint-disable selector-max-compound-selectors, max-nesting-depth
+// stylelint-disable declaration-property-value-blacklist scss/at-mixin-pattern
+// stylelint-disable scss/at-mixin-pattern
+
 :local {
-  .date-picker {
+  @mixin focus-styles {
+    border-color: var(--terra-date-picker-focus-border-color, #26a2e5);
+    border-style: var(--terra-date-picker-focus-border-style, solid);
+    border-width: var(--terra-date-picker-focus-border-width, 1px);
+    opacity: var(--terra-date-picker-focus-opacity, 1);
+    outline: none;
```

Comment:
We need to update both themes for these changes: https://github.com/cerner/terra-framework/tree/master/packages/terra-date-picker/themes